maintainers/maintainer-list.nix

```diff
@@ -7758,6 +7758,12 @@
     githubId = 1711539;
     name = "matklad";
   };
+  matrss = {
+    name = "Matthias Riße";
+    email = "matrss@users.noreply.github.com";
```

Comment:
Can you use a none github mail?

Reply:
Sure, I will put my primary mail address for github there for now. I might want to change that to an address on my own domain in the future though, just did not get around to setting that up yet. I assume I can just change it in the future, when I have a PR for something else anyway?

Reply:
yeah, you can change it later.